packages/dusk-cli/src/lib/extractMultiplayerMetadata.ts

```diff
@@ -6,6 +6,7 @@ const regexes = {
   updatesPerSecondDefined: /updatesPerSecond\s*:/,
   updatesPerSecond: /updatesPerSecond\s*:\s*?([0-9]+)/,
   inputDelay: /inputDelay\s*:\s*?([0-9]+)/,
+  landscape: /landscape\s*:\s*?(true|false)/,
```

Comment:
@Gongreg are we going to see the same issue with !0 after minification here that was on the persistence property? (I think you turned it off for Pinpoint?)

Reply:
We are going to see the same issue.

Reply:
We could, but I keep the approach [for consistency](https://github.com/rune/rune-multiplayer-web-games/pull/382#discussion_r1594712728) unless we want to change all the other fields as well now.